mdtraj/formats/pdb/pdbfile.py

```diff
@@ -332,7 +332,10 @@ def write(self, positions, topology, modelIndex=None, unitcell_lengths=None,
         if modelIndex is not None:
             print("MODEL     %4d" % modelIndex, file=self._file)
         for (chainIndex, chain) in enumerate(topology.chains):
-            chainName = self._chain_names[chainIndex % len(self._chain_names)]
+            if chain.chain_id == None or chain.chain_id == '':
+                chainName = self._chain_names[chainIndex % len(self._chain_names)]
+            else:
+                chainName = chain.chain_id[0:1]
```

Comment:
can be simplified as both `None` and `''` evaluate to `False`
```suggestion
            if not chain.chain_id:
                chainName = self._chain_names[chainIndex % len(self._chain_names)]
            else:
                chainName = chain.chain_id[0:1]
```

Reply:
you're right, thanks. Done.